docs/wiki/feature/pylons.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: wiki
+title: Pylons
+description: Adds an interface that allow players to configure aircraft pylons.
+group: feature
+category: interaction
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 11
+  patch: 0
+---
+
+## 1. Overview
+
+Adds an interface that allow players to configure aircraft pylons. Loadouts can be saved and loaded through interface.
+
+## 2. Usage
+
+### 2.1 Configure Pylons
+
+To configure pylons you need a `Toolkit` equipped.
+
+- Interact with aircraft using <kbd>⊞ Win</kbd> (ACE3 default key bind `Interaction Key`).
+- Select `Configure Pylons`.
+- Configure your loadout in the menu that appears.
+- Press `APPLY` when done.
+- Rearm the aircraft (See [Rearm Wiki](rearm.md)).
```

Comment:
Lower-case `see`.

Reply:
You can turn off the need to do step 5, it's a setting.